api/app/Models/Pool.php

```diff
@@ -541,43 +541,47 @@ public static function scopeClassifications(Builder $query, ?array $classificati
     }
 
     /**
-     * Filter for pools the user is allowed to view admin information for, based around assessment plan permissions
+     * Filter for pools the user is allowed to admin, based on scopeAuthorizedToAdmin
      */
-    public static function scopeAuthorizedToViewAsAdmin(Builder $query, ?bool $canAdmin): Builder
+    public static function scopeCanAdmin(Builder $query, ?bool $canAdmin): Builder
     {
-        if (empty($canAdmin)) {
-            return $query;
+        if (! empty($canAdmin)) {
```

Comment:
This maintains the existing behaviour - but maybe it should be `if ($canAdmin)` so it only runs if truthy?

Reply:
Oh, interesting!  I agree that if it confused us both it should change.
[clarify scopeCanAdmin test](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11138/commits/c81a8bc0e1d77913b9a145617ef493bf640dc555)